mmrazor/models/mutables/mutable_channel/mutable_channel.py

```diff
@@ -87,7 +87,27 @@ def bind_mutable_name(self, name: str) -> None:
         Args:
             name (str): Name of this `MutableChannel`.
         """
-        self.name = name
+        self._name = name
+
+    def bind_mutable_type(self, mutable_type: str) -> None:
```

Comment:
The name `bind_mutable_type` is a bit confusing, according to the reference in [`bind_mask_type`](https://github.com/HIT-cwh/mmrazor/blob/fe6873c70f77d5731f7abd0ddda514f97b9b5f08/mmrazor/models/mutators/channel_mutator/channel_mutator.py#L172), it's more like a module name but not a type of mutable.